### PR TITLE
#2480 - Align user avatar in the post composer correctly

### PR DESCRIPTION
**Pull request by gurmeetb**
Fix for #2480

### CODE REVIEW

**Comment by gurmeetb** [comment]
@maxceem i see we have component (Avatar) coming from other repo - `appirio-tech-react-components` - I don't have access to the same. Could you guide me how do I get handle over it ?

File: src/components/RichTextArea/RichTextArea.jsx [hunk 1]
```diff
@@ -303,7 +303,7 @@ class RichTextArea extends React.Component {
         <div className="modal-row">
           {avatarUrl &&
             <div className="portrait">
-              <Avatar avatarUrl={getAvatarResized(avatarUrl, 30)} userName={authorName} />
+              <Avatar size={40} avatarUrl={getAvatarResized(avatarUrl, 30)} userName={authorName} />
```
Comment (maxceem) [inline src/components/RichTextArea/RichTextArea.jsx]: It's almost done. But the image by itself is still 30x30 so it still smaller than avatars of comments.  Most likely function `getAvatarResized` has to be called with the second param as `40` to generate a URL for image 40x40.

**Comment by gurmeetb** [comment]
@maxceem Kindly validate☝️

**Comment by maxceem** [comment]
@gurmeetb the image now comes in a proper size, but this change is not enough to make image same size, see

image

Please, make all neccesary changes so that avatar is displayed in a proper size here.

**Comment by gurmeetb** [comment]
is it fine if we show tooltip too? else we might need to replicate bit of logic from UserToolTip here - i did try out with it and it comes fine - only issue being it comes with tooltip

**Comment by maxceem** [comment]
I think we don't need a tooltip here, and we can implement it without using a tooltip. But let me ask first if we can add a tooltip for this case.

@RishiRajSahu I think we shouldn't show a tooltip for the avatar of our own user here. Please, let me know if you think we may need it here.

**Comment by RishiRajSahu** [comment]
@maxceem yeah tooltip for avatar beside post-composer is not needed.

**Comment by maxceem** [comment]
@gurmeetb 

> will need to tweak the logic a bit.

I'm not sure if it really needs to tweak the logic here. As I see it adjusting or adding some CSS styles has to be enough. Maybe adding some className to some component if really necessary.

Do you have some difficulties to adjust the size only by CSS styles? Let me know if you do, maybe I could help with some ideas.

**Comment by gurmeetb** [comment]
@RishiRajSahu thanks for confirming.

@maxceem let me get back on this in few hrs, will let know if i get stuck. I will look into this in 1-2 hrs, moved out for some work - away from computer right now..let me get back asap

**Comment by maxceem** [comment]
@gurmeetb I cannot find any user I know without an avatar, but you can log out and after register a new user using your e-mail. So it will be a new user on the DEV environment.

**Comment by gurmeetb** [comment]
Thanks @maxceem for Firefox Inspector help. It was quick to find where I was going wrong. It is fixed and pushed in now. 

Thanks :)

**Pull request merged**